views/part-discount-banner.php

```diff
@@ -6,9 +6,10 @@
 	<p class="imagify-promo-title">
 		<?php
 		printf(
-			/* translators: %s is a formatted percentage. */
-			__( '%s OFF on all the subscriptions', 'imagify' ),
-			'<span class="imagify-promotion-number"></span>'
+			/* translators: First %s is a formatted percentage, Second %s is a subscription plan name. */
+			__( '%s OFF on %s subscriptions', 'imagify' ),
```

Comment:
```suggestion
			__( '%1$s OFF on %2$s subscriptions', 'imagify' ),
```

Little improvement to always match the placeholders with the arguments order